### DISCUSSION TITLE
How do you select things in motion state?

### DISCUSSION THREAD

**Discussion by brabalan** [meow-edit/meow#569] [Q&A] [answered]
I’m using motion state for `notmuch-show-mode`, but I sometimes need to select select some text (typically once a week when I collect the contents of some emails for a newsletter). At the moment I use `C-space` to start a plain emacs selection, but I would rather use meow’s bindings. Being in normal mode by default does not seem to be a solution as I would lose all the default notmuch keybindings… How would you do it?

**Comment by DogLooksGood** [comment, marked as answer]
I suggest use emacs default key bindings. Since we are using Emacs, it's better to get used to the builtins.

But there's also a command `meow-temp-normal` to let you enter normal state temporarily, then use `meow-insert` to switch back to motion state.
